File: app/src/main/java/can/siempredelao/f1kotlin/backend/QualifyingResult.java
package can.siempredelao.f1kotlin.backend;

import com.google.gson.annotations.SerializedName;

public class QualifyingResult {

    public String      number;
    public String      position;
    @SerializedName("Driver")
    public Driver      driver;
    @SerializedName("Constructor")
    public Constructor constructor;
    @SerializedName("Q1")
    public String      q1;
    @SerializedName("Q2")
    public String      q2;
    @SerializedName("Q3")
    public String      q3;
}

File: app/src/main/kotlin/can/siempredelao/f1kotlin/backend/model/QualifyingResult.kt
package can.siempredelao.f1kotlin.backend.model

import com.google.gson.annotations.SerializedName

class QualifyingResult(val number: String,
                       val position: String,
                       @SerializedName("Driver") val driver: Driver,
                       @SerializedName("Constructor") val constructor: Constructor,
                       @SerializedName("Q1") val q1: String,
                       @SerializedName("Q2") val q2: String,
                       @SerializedName("Q3") val q3: String)
